### PR TITLE
Use full location URL for xlink references in SVG <use> elements

### PR DESCRIPTION
**Pull request by dpvc**
Use full location URL for xlink references in SVG `<use>` elements when the page includes a `<base>` element.  Resolves issue #831.

The might cause issues when the page URL uses query parameter (i.e., ends with `?...`).  For situations where that will occur, the use of `useFontCase: false` is recommended.

### CODE REVIEW

File: unpacked/jax/output/SVG/jax.js [hunk 1]
```diff
@@ -33,6 +33,13 @@
   var SVGNS   = "http://www.w3.org/2000/svg";
   var XLINKNS = "http://www.w3.org/1999/xlink";
 
+  //
+  //  Get the URL of the page (for use with xlink:href) when there
+  //  is a <base> element on the page.
+  //
```
Comment (zorkow) [inline unpacked/jax/output/SVG/jax.js]: "when there is >no< <base> element on the page"?

File: unpacked/jax/output/SVG/jax.js [hunk 2]
```diff
@@ -33,6 +33,13 @@
   var SVGNS   = "http://www.w3.org/2000/svg";
   var XLINKNS = "http://www.w3.org/1999/xlink";
 
+  //
+  //  Get the URL of the page (for use with xlink:href) when there
+  //  is a <base> element on the page.
+  //  
+  var SVGURL = String(document.location).replace(/#.*$/,"");
+  if (document.getElementsByTagName("base").length === 0) SVGURL = "";
+
```
Comment (zorkow) [inline unpacked/jax/output/SVG/jax.js]: var SVGURL = (document.getElementsByTagName("base").length === 0) ? "" : String(document.location).replace(/#.*$/,"");

will make it more efficient.

File: unpacked/jax/output/SVG/jax.js (hunk 1 above)
Comment (dpvc) [inline unpacked/jax/output/SVG/jax.js]: No, it is when there is a `<base>` element, as the comment says now.  The issue is that the `<base>` element makes relative xlink references point to the wrong place (the base), so we need to use absolute references in that case (the case where there is a `<base>` element).

File: unpacked/jax/output/SVG/jax.js (hunk 2 above)
Comment (dpvc) [inline unpacked/jax/output/SVG/jax.js]: OK.

File: unpacked/jax/output/SVG/jax.js (hunk 1 above)
Comment (zorkow) [inline unpacked/jax/output/SVG/jax.js]: Right, got it.

**Comment by dpvc** [comment]
I've made the suggested change.

**Comment by zorkow** [comment]
lgtm.

**Pull request merged**